src/Forms/Shared/Samples/Map/BrowseBuildingFloors/readme.md

```diff
@@ -0,0 +1,43 @@
+# Browse building floors
+
+Display and browse through building floors from a floor-aware web map.
+
+![BrowseBuildingFloorsApp](browse-building-floors.png)
+
+## Use case
+
+Having map data to aid indoor navigation in buildings with multiple floors such as airports, museums, or offices can be incredibly useful. For example, you may wish to browse through all available floor maps for an office in order to find the location of an upcoming meeting in advance.
+
+## How to use the sample
+
+Use the spinner to browse different floor levels in the facility. Only the selected floor will be displayed.
```

Comment:
We don't use a "spinner" control on WPF. You should change this to a more generic term that applies to each version of the sample.